package.json

```diff
@@ -45,7 +48,10 @@
     },
     "license": "BSD-2-Clause",
     "jest": {
-        "preset": "ts-jest"
+        "preset": "ts-jest",
```

Comment:
Is there a way to test the ESM version of the module as well? Eg. if a `__dirname` was missed somewhere — would that be caught?